### ISSUE TITLE
a question on the standalone player

### ISSUE THREAD

**Issue by ousia** [ruffle-rs/ruffle#1761]
@Toad06,

since you provided #1161 with options for autoplay, I have an improvement suggestion about it.

I have used the standalone version in https://www.free-culture.tk/fc.html, with autoplay disabled.

The actual animation takes 8.9MB. I think it would make sense that it is downloaded only after the user clicks to play it.

My proposal is to avoid unnecessary data waste. What do you think about it?

Many thanks for your help.

**Comment by Toad06** [comment]
Autoplay has been implemented in #1556, not in #1161, so the options are different:
```
window.RufflePlayer.config = {
	autoplay: "val", // (val can be "auto" - the default value, Ruffle checks if the browser supports autoplay - "off" or "on" -)
	unmuteOverlay: "val" (val can be "visible" or "hidden" - this option is only useful if autoplay is "on" AND the browser doesn't support autoplay)
};
```

If you want the animation to be downloaded after the user clicks it, I recommend using Javascript. For instance:

In the head tag, add:
```
<script>
function playOnClick() {
	const container = document.getElementsByClassName("flash-presentation")[0];
	container.innerHTML = "";
	const ruffle = window.RufflePlayer.newest();
	const player = ruffle.createPlayer();
	player.style.width = "100%";
	player.style.height = "600px";
	player.config.autoplay = "auto";
	container.appendChild(player);
	player.load("./free_culture.swf");
}
</script>
```

In your "flash-presentation" div, replace the object tag with:
`<a href="#" onclick="playOnClick(); return false;">Load file!</a>`

**Comment by ousia** [comment]
@Toad06, many thanks for your reply.

This is exactly what I needed.
